src/Form/AppSettingsForm.php

```diff
@@ -90,6 +91,7 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#type' => 'checkbox',
       '#title' => $this->t('Allow selecting multiple products'),
       '#default_value' => $generalConfig->get('multiple_products'),
+      '#disabled' => !($this->config('apigee_edge.dangerzone')->get('skip_developer_app_settings_validation')) && (bool) DeveloperApp::loadMultiple(),
```

Comment:
Do not load all developer apps as object unnecessary. Just get the list of developer app *ids*.

Reply:
Please also add a comment to this line that "dangerzone variable" check was only needed to make tests pass.